app/bundles/EmailBundle/EventListener/EmailSubscriber.php

```diff
@@ -187,17 +187,26 @@ public function onEmailParse(Events\ParseEmailEvent $event)
         // Listening for bounce_folder and unsubscribe_folder
         $isBounce      = $event->isApplicable('EmailBundle', 'bounces');
         $isUnsubscribe = $event->isApplicable('EmailBundle', 'unsubscribes');
-
+        $isReply       = $event->isApplicable('EmailBundle', 'reply');
+        /** @var \Mautic\EmailBundle\Helper\MessageHelper $messageHelper */
+        $messageHelper = $this->factory->getHelper('message');
```

Comment:
Lets not use MauticFactory in the new code since it had been deprecated. Please add the MessageHelper as a new dependency to this PHP class.

Reply:
i have replace the usage of factory. Please check.